personal_finances/merge_transactions.py

```diff
@@ -73,6 +73,12 @@ def merge_transactions() -> None:
         f"data/merged_transactions-{datetime.now().isoformat()}.json",
         deduped_transactions,
     )
+    # Issue 37:
+    # Generate last merged_transactions_latest.json due filename change
```

Comment:
I suggest adding comments only if there's something non-trivial, that can't be refactored, for understanding what the code does.

We can get tracking info from git commits + PRs